content/blog/2019/secure-egress-control-istio/index.md

```diff
@@ -0,0 +1,323 @@
+   enforcement of the security policies.
+   This attack is prevented by applying the special security measures to the egress gateway pod.
+1. Since the previous attacks are prevented, the attackers have no other option but to direct the traffic through the
+   egress gateway. The traffic will be monitored by the egress gateway, so the goal of the attackers to access
+   external services unmonitored cannot be achieved. The attackers may want to try to achieve their second goal, that is
+   to access `mongo1.composedb.com`. To achieve it, they may try to **impersonate** as the application B since the
+   application B is allowed to access `mongo1.composedb.com`. This attack, fortunately, is prevented by Istio's [strong
+   identity support](/docs/concepts/security/#istio-identity).
+
+After I showed how Istio egress traffic control can prevent possible attacks, let me describe its advantages over
+Kubernetes Network Policies and legacy egress proxies/firewalls.
+
+### Advantages of Istio egress traffic control
+
+Istio egress traffic control is **DNS-aware**: you can define policies based on URLs or on wildcard domains like
+`*.ibm.com`. In this sense, it is better than Kubernetes Network Policies which are not DNS-aware.
+
+Istio egress traffic control is **transparent** with regard to TLS traffic, since Istio is transparent:
+you do not need to change the applications or to configure their containers.
+For HTTP traffic with TLS origination, DevOps people must configure the applications to use HTTP instead of HTTPS
+when the applications run with Istio sidecars injected.
+
+Istio egress traffic control is **Kubernetes-aware**: the identity of the source of egress traffic is based on
+Kubernetes service accounts. Istio egress traffic control is better than the legacy DNS-aware proxies/firewalls which
+are not transparent and not Kubernetes-aware.
+
+Istio egress traffic control is **secure**: it is based on the strong identity of Istio and, when the cluster operators
+provide
+[additional security measures](/docs/examples/advanced-gateways/egress-gateway/#additional-security-considerations),
+it is tampering-proof.
+
+On top of these beneficial features, Istio egress traffic control provides additional advantages:
+
+*  It allows defining access policies in the same language for ingress/egress/in-cluster traffic. The cluster operators
+   need to learn a single policy and configuration language for all the kinds of traffic.
+*  Istio egress traffic control is integrated with Istio policy and telemetry adapters and can work out-of-the-box.
+*  When you use external monitoring/access control systems with Istio, you must write the adapters for them only once,
+   and then apply the adapters for all the kinds of traffic: ingress/egress/in-cluster.
+*  The Istio operators can apply Istio traffic management features to egress traffic, such as
```

Comment:
The Istio -> Istio

Reply:
Fixed in https://github.com/istio/istio.io/pull/3179/commits/2dbbb63debb55b0a072fd399271fea5ea89d145a